### PR TITLE
RPN-580

### PR DESCRIPTION
**Pull request by mystdeim**

### CODE REVIEW

File: nft/core/src/test/kotlin/com/rarible/protocol/nft/core/service/item/meta/ItemPropertiesServiceTest.kt [hunk 1]
```diff
@@ -195,6 +195,24 @@ Token ID: 51561
         assertThat(properties.attributes).contains(ItemAttribute("Mouth", "Phoneme Vuh"))
     }
 
+    @Test
+    internal fun etherCats() {
+        val properties = testing.getProperties(Address.apply("0xff3559412c4618af7c6e6f166c74252ff6364456"), BigInteger.valueOf(50101)).block()!!
+        assertEquals("Gatinho [10 Votes, Common]", properties.name)
+        assertEquals("This Chainlink VRF common Gatinho purrs with a rating of 10, and a multiplier of 1x. It has a total score of 10, and confers the same amount of votes in the EtherCats DAO. The card game properties are 8 for North, Stringalong for East, Box for South, and 1 for West. Gatinho has the personality suit of punter. Each Founders Series cat is part of NFT history. These fine felines represent the first verifiably random packs minted with Chainlink VRF.", properties.description)
```
Comment (yakischik) [inline nft/core/src/test/kotlin/com/rarible/protocol/nft/core/service/item/meta/ItemPropertiesServiceTest.kt:202]: I think we should not check such large values in tests

File: nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/meta/descriptors/Helpers.kt [hunk 2]
```diff
@@ -17,7 +17,7 @@ fun String.parseTokenId(): Pair<Address, BigInteger> {
 fun JsonNode.getText(vararg paths: String): String? {
     for (path in paths) {
         val current = this.path(path)
-        if (current.isTextual) {
+        if (current.isTextual || current.isNumber) {
```
Comment (yakischik) [inline nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/meta/descriptors/Helpers.kt:20]: Boolean?

**Review by yakischik — APPROVED** [review]

File: nft/core/src/test/kotlin/com/rarible/protocol/nft/core/service/item/meta/ItemPropertiesServiceTest.kt (hunk 1 above)
Comment (mystdeim) [reply nft/core/src/test/kotlin/com/rarible/protocol/nft/core/service/item/meta/ItemPropertiesServiceTest.kt:202]: What do you suggest to do in this case?

File: nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/meta/descriptors/Helpers.kt (hunk 2 above)
Comment (mystdeim) [reply nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/meta/descriptors/Helpers.kt:20]: Haven't met boolean values in meta yet...

File: nft/core/src/test/kotlin/com/rarible/protocol/nft/core/service/item/meta/ItemPropertiesServiceTest.kt (hunk 1 above)
Comment (yakischik) [reply nft/core/src/test/kotlin/com/rarible/protocol/nft/core/service/item/meta/ItemPropertiesServiceTest.kt:202]: Description is not really important business-field, maybe here we can use something like contains() and part of description as arg

File: nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/meta/descriptors/Helpers.kt (hunk 2 above)
Comment (yakischik) [reply nft/core/src/main/kotlin/com/rarible/protocol/nft/core/service/item/meta/descriptors/Helpers.kt:20]: Ok

**Pull request merged**